package.json

```diff
@@ -58,7 +58,7 @@
     "bignumber.js": "^9.0.2",
     "content-hash": "^2.5.2",
     "copy-to-clipboard": "^3.3.1",
-    "dxdao-contracts": "https://github.com/DXGovernance/dxdao-contracts.git#develop",
+    "dxdao-contracts": "https://github.com/DXGovernance/dxdao-contracts.git#12ea7f4e1ea070699d148d8c4ced4aaa88f08627",
```

Comment:
It is better to point to a specific commit and update the dependecy with versions or commits rather than pointing a branch that can introduce changes without testing the integration beforehand.